src/com/videojs/providers/HLSProvider.as

```diff
@@ -233,8 +233,9 @@ package com.videojs.providers{
 
           for (var i: uint = 0; i < captionData.length; i++) {
             external.push({
-              pos: captionData[i].pos,
```

Comment:
Based on incoming changes to https://github.com/brightcove/flashls/pull/7 this should be added back in.